index.html

```diff
@@ -0,0 +1,1543 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Twitter</title>
+</head>
+
+<body>
```

Comment:
suggest using auto-formatter to tweak some of the div nesting - in some cases it's slightly off.

You can configure VSCode to automatically run the formatter when you save or paste - this can trigger the Prettier extension.